wordlist.txt

```diff
@@ -135,6 +138,7 @@ rlp
 rlpx
 RLPx
 rpc
+sanaz
```

Comment:
Huh, surprised it is lowercase, does that work? I thought the spell checker was case sensitive

Reply:
Apparently yes!